packages/metronome/buildinfo.json

```diff
@@ -2,8 +2,8 @@
   "requires": ["java", "exhibitor"],
   "single_source": {
     "kind": "url_extract",
-    "url": "https://downloads.mesosphere.com/maven/dcos/metronome/0.2.3/metronome-0.2.3.tgz",
-    "sha1": "ef42cb936a59360ecd0dda3a434f75e1007f3527"
+    "url": "http://downloads.mesosphere.io/metronome/snapshots/metronome-0.2.4.tgz",
```

Comment:
Just curious why it ended up in the `snapshots` directory?

Reply:
this was made to be the same as marathon.   we need to fix our release process.